snowdrop-block/src/Snowdrop/Block/State.hs

```diff
@@ -87,76 +78,45 @@ mapTxs_ handleTxDo txsWithAccs =
 -- It uniformly accesses state and block storage (via `DataAccess` interface).
 inmemoryBlkStateConfiguration
   :: forall blkType txtypes conf xs c m .
-    ( xs ~ BlkProcComponents blkType txtypes c
+    ( xs ~ BlkProcComponents blkType txtypes
     , c ~ BlkProcConstr txtypes xs
     , m ~ ERwComp conf (DbAccessU conf xs) (ChgAccum conf)
+    , Tx blkType ~ SomeTx (BlkProcConstr txtypes xs)
     , HasBExceptions conf
         [ StatePException
         , CSMappendException
         ]
-    , Payload blkType ~ [SomeTx c]
+    , Default (ChgAccum conf)
+    , Default (SumChangeSet (UnionSeqExpandersInps txtypes))
+    , HasGetter (BlockUndo blkType) (HChangeSet xs)
```

Comment:
This constraint is wrong, this won't be true for AVL+ trees

Reply:
See other comment, for `bscInmemRollback`, constraint `HasGetter (Undo conf) (BlockUndo blkType)` seems to be needed here instead.